show/bgp_quagga_v4.py

```diff
@@ -1,6 +1,71 @@
 import click
+import json
+import re
+from natsort import natsorted
+from tabulate import tabulate
+from natsort import natsorted
+from tabulate import tabulate
 from show.main import *
 
+def parse_bgp_summary(summ):
```

Comment:
>def [](start = 0, length = 3)

Suggest move command functions to [https://github.com/Azure/sonic-py-swsssdk](https://github.com/Azure/sonic-py-swsssdk), so sonic-snmpagent can also access.

Reply:
Good idea, will sync with Joe on this.